db/python/tables/family_participant.py

```diff
@@ -159,6 +159,50 @@ async def get_rows(
 
         return ds
 
+    async def get_row(
+        self,
+        family_id: int,
+        participant_id: int,
+    ):
+        """Get a single row from the family_participant table"""
+        keys = [
+            'fp.family_id',
+            'p.id as individual_id',
+            'fp.paternal_participant_id',
+            'fp.maternal_participant_id',
+            'p.reported_sex as sex',
+            'fp.affected',
+        ]
+        keys_str = ', '.join(keys)
+
+        values: Dict[str, Any] = {
+            'family_id': family_id,
+            'participant_id': participant_id,
+        }
+
+        wheres = ['f.id = :family_id', 'p.id = :participant_id']
+        conditions = ' AND '.join(wheres)
+
+        _query = f"""
+            SELECT {keys_str} FROM family_participant fp
+            INNER JOIN family f ON f.id = fp.family_id
+            INNER JOIN participant p on fp.participant_id = p.id
+            WHERE {conditions}"""
```

Comment:
Small nitpick, but if you're not dynamically changing the conditions and the keys are static to this method, you can just include them as part of the query and remove the fstring